x-pack/plugins/security_solution/tsconfig.json

```diff
@@ -43,6 +43,7 @@
     { "path": "../spaces/tsconfig.json" },
     { "path": "../security/tsconfig.json" },
     { "path": "../timelines/tsconfig.json" },
+    { "path": "../../../src/plugins/unified_search/tsconfig.json" },
```

Comment:
Better to  move this up below [line 30](https://github.com/elastic/kibana/pull/127651/files#diff-844dde79a96b9e6c2bfdf21f8c80feb8b687577e21326da6bd15c6b95d8ec918L30)